setup.sh

```diff
@@ -511,19 +511,6 @@ then
 		export MACOS_UNIVERSAL=no
 	fi
 
-	# remove angr-management if running in pypy or in travis
-	#(python --version 2>&1| grep -q PyPy) && 
-	info "NOTE: removing angr-management until we sort out the pyside packaging"
-	TO_INSTALL=${TO_INSTALL// angr-management/}
-	info "Install list: $TO_INSTALL"
-	[ -n "$TRAVIS" ] && TO_INSTALL=${TO_INSTALL// angr-management/}
-
-    	for PACKAGE in $TO_INSTALL; do
```

Comment:
Are you sure you wanted to remove this stanza? this is where $TO_INSTALL is actually processed.

Reply:
oh whoops, thank you!